deploy/osd-17664-reinstall-mgo/pre-4.11/config.yaml

```
@@ -0,0 +1,11 @@
+---
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  resourceApplyMode: Sync
+  matchExpressions:
+  - key: hive.openshift.io/version-major-minor
+    operator: In
+    values: ["4.6","4.7","4.8","4.9","4.10"]
```

```suggestion
    values: ["4.10"]
```

Do we want to touch our unsupported clusters?